src/Assignment-2/assignment2.html

```diff
@@ -10,12 +10,162 @@
 
 <body>
 	<div id="container">
-
+		
+		<header style="border-style: solid; max-height: 100px; height: 100px;">
+			<div class="headerimg">
+			  <img src="https://images.unsplash.com/photo-1599305445671-ac291c95aaa9?ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8bG9nb3xlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60">
+			</div>
+		
+			<div class="headerinput" >
```

Comment:
did not use form tag here.
![image](https://user-images.githubusercontent.com/92850233/140770363-55b70aa9-59bd-41b3-8845-ea1975d02aa0.png)